min-turnup/phase2/ansible/playbooks/roles/master/templates/kubeconfig.jsonnet

```diff
@@ -1,26 +0,0 @@
-function(cfg)
```

Comment:
I have no idea why this is showing up like this. In both the upstream branch and my own branch, the file `kubeconfig.jsonnet` lives at `roles/node/templates`. I can't explain why GitHub is showing this diff...